tests/test_config.py

```diff
@@ -18,7 +19,7 @@ def setUp(self):
         self.config = Config()
 
     def test_no_config(self):
-        self.assertRaises(SystemExit, Config)
+        self.assertRaises(SystemExit, patroni_main)
```

Comment:
That's weird, we should not test the `patroni_main()` here

Reply:
moved to test_patroni.py